### PR TITLE
Update 3-pooling.mdx

### PR DESCRIPTION
**Pull request by nyurik**
separate promise and await/async examples for clarity

### CODE REVIEW

File: content/features/3-pooling.mdx [hunk 1]
```diff
@@ -65,19 +65,20 @@ pool
...
-      } finally {
-        client.release()
-      }
-    }
-  )()
-  .catch(e => console.log(e.stack))
+  })
+
+// async/await - check out a client
+(async () => {
```
Comment (charmander) [inline content/features/3-pooling.mdx]: The original reason reformatting combined them is that this is an ASI reliance mistake when it’s one script.

```suggestion
;(async () => {
```

Comment (charmander) [reply content/features/3-pooling.mdx]: Formatted like that, really: https://github.com/brianc/node-postgres-docs/commit/f38a3ab17959e9f5cb22d04d34a5ae8f75a839fd

Comment (nyurik) [reply content/features/3-pooling.mdx]: thanks, fixed

File: content/features/3-pooling.mdx [hunk 2]
```diff
@@ -65,19 +65,20 @@ pool
...
+  const client = await pool.connect()
+  try {
+    const res = await client.query('SELECT * FROM users WHERE id = $1', [1])
+    console.log(res.rows[0])
+  } finally {
+    // Make sure to release the client before any error handling,
+    // just in case the error handling itself throws an error.
+    client.release()
+  }
+})().catch(e => console.log(e.stack));
```
Comment (charmander) [inline content/features/3-pooling.mdx]: in addition to the placement from the last comment, no semicolon needs to be added here:

```suggestion
})().catch(e => console.log(e.stack))
```

File: content/features/3-pooling.mdx (hunk 1 above)
Comment (charmander) [reply content/features/3-pooling.mdx]: It goes before the `(`, rather than on the preceding line.

Comment (nyurik) [reply content/features/3-pooling.mdx]: oh, that's a weird semicolon placement. I noticed you don't like semicolons much :)  Will change.

Comment (charmander) [reply content/features/3-pooling.mdx]: Yep, it’s weird! I do like semicolons, but the pg style is to omit them.

**Comment by nyurik** [comment]
@charmander anything else I need to do for this PR? thx!

**Comment by charmander** [comment]
@nyurik Not that I can see! @brianc should be around to merge it at some point.

**Comment by brianc** [comment]
Sorry for the delay! Thanks for the docs update! ❤️

**Pull request merged**